client/worker.go

```diff
@@ -52,16 +52,21 @@ func newWorker(client *http.Client, endpoint string, accessToken string) *worker
 	return w
 }
 
-func (w *worker) export(ctx context.Context, batch *jaegerpb.Batch) *ErrSend {
+func (w *worker) export(ctx context.Context, batches []*jaegerpb.Batch) *ErrSend {
 	_, span := trace.StartSpan(ctx, "export")
 	defer span.End()
 
-	span.AddAttributes(trace.Int64Attribute("spans", int64(len(batch.Spans))))
-	if len(batch.Spans) == 0 {
+	var spansCount int
+	for _, batch := range batches {
+		spansCount += len(batch.Spans)
+	}
+
+	span.AddAttributes(trace.Int64Attribute("spans", int64(spansCount)))
+	if spansCount == 0 {
```

Comment:
Can we add a `batches` count to the span as well?

Reply:
added